tests/local_ping_echo.sh

```diff
@@ -21,6 +21,21 @@ $BINS/examples/dtnping -d 'dtn://node1/echo' -c 6 -t 500ms
 RC=$?
 echo "RET: $RC"
 
+echo -n "Bundles in store on node 1: "
+NUM_BUNDLES=$($BINS/dtnquery bundles | grep "dtn://" | wc -l | awk '{print $1}')
+echo -n $NUM_BUNDLES
+
+EXPECTED_BUNDLES=0
```

Comment:
just to be sure you should also check for deleted bundles in `dtnquery store` otherwise it would also succeed if no bundles where received at all.

Reply:
does the test still ensure that bundles where indeed delivered? did you deactivate the dtnecho service and see if it properly fails?